JetStreamDriver.js

```diff
@@ -986,6 +1007,94 @@ class AsyncBenchmark extends DefaultBenchmark {
     }
 };
 
+// Meant for wasm benchmarks that are directly compiled with an emcc build script. It might not work for benchmarks built as
+// part of a larger project's build system or a wasm benchmark compiled from a language that doesn't compile with emcc.
+class WasmEMCCBenchmark extends AsyncBenchmark {
```

Comment:
LGTM for now, long term it would be nice to get rid of some of the code duplication, e.g., the `prerunCode` below is mostly (but not fully) equal to the one in `WasmLegacyBenchmark` right?

Reply:
Yeah, I think the goal is to eventually just delete WasmLegacyBenchmark once we have all the benchmarks we care about in the JS style system. If we want to save them for posterity we can reevaluate.